pandas/core/indexes/multi.py

```diff
@@ -896,17 +896,23 @@ def _get_level_values(self, level):
         Parameters
         ----------
         level : int level
+        unique : bool
+            if True, drop duplicated values
+
+            .. versionadded:: 0.21.0
```

Comment:
this is not needed for an internal method IMO

Reply:
ah, I see that actually @jreback asked to add it above. But still, I don't think this should be added for internal private functions, as internal code should never have to care about the pandas version